src/cmd/services/m3aggregator/config/config.go

```diff
@@ -26,13 +26,32 @@ import (
 	"github.com/m3db/m3/src/x/log"
 )
 
+var (
+	defaultMetricsSanitization        = instrument.PrometheusMetricSanitization
```

Comment:
Where do these defaults come from?

Reply:
Copied from the dbnode test. 
I reverted this change and added the metrics config in the test yaml again.
I can work on the default values later in separate PRs.